web/client/codechecker_client/cmd/cmd.py

```diff
@@ -1492,6 +1507,17 @@ def add_arguments_to_parser(parser):
     importer.set_defaults(func=cmd_line_client.handle_import)
     __add_common_arguments(importer)
 
+    labels = subcommands.add_parser(
+        'labels',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        description="Checkers can be assigned properties by labels. This "
+                    "command lists these and store them to the server so "
+                    "filtering is possible based on these labels.",
```

Comment:
Are you sure that this is true? I think now we can't store these labels to the server and we can't filter on them.